src/bsp/platform/cc3200/xi_bsp_hton.h

```diff
@@ -0,0 +1,2 @@
+#include <socket.h>
+typedef long int ssize_t;
```

Comment:
hrm.. I don't think I have anything that comes to mind as an elegant way to solve this problem. :(